web/src/PulseGuide.jsx

```
@@ -0,0 +1,104 @@
+    <>
+      <button
+        className={`pulse-guide-fab ${isOpen ? 'open' : ''} ${isHovered ? 'hovered' : ''}`}
+        onClick={() => setIsOpen(!isOpen)}
+        onMouseEnter={() => setIsHovered(true)}
+        onMouseLeave={() => setIsHovered(false)}
+        aria-label="Pulse Guide Assistant"
+        title="Pulse Guide"
+      >
+        <div className="pulse-guide-icon">
+          {isOpen ? (
+            <svg width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" strokeWidth="2" strokeLinecap="round" strokeLinejoin="round">
```

Biome (2.3.13)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)